modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -46,31 +31,31 @@ ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
 nameserver=4.4.4.41
 ----
 
-a|Specify multiple network interfaces by specifying multiple `ip=` entries.
-
+|Specify multiple network interfaces by specifying multiple `ip=` entries.
```

Comment:
Not sure why these were dropped. The `a` is needed in some of these cells to properly display formatting.